routes/users.js

```diff
@@ -222,6 +221,11 @@ router.post('/signup', signupValidationRules(), validate, async (req, res) => {
   if (candidate) {
     return sendError(res, 409, 'This email is already taken. Try another');
   } else {
+    if (!password) return sendError(res, 206, { message: 'Password missing' });
```

Comment:
this should not be in here

Reply:
validation is now handled elsewere and I removed it a while ago

Reply:
:thinking: It was from master hmm..

Reply:
if you compare with current master they are not there